src/main/resources/templates/macros/report/message.vm

```diff
@@ -5,11 +5,11 @@
 
   <div class="inner-level">
     <div class="message indention">
-      <div data-toggle="collapse" class="collapsable-control" data-target="#msg-$msgId">
-          <a>$messageName</a>
+      <div data-toggle="collapse" class="collapsable-control" data-target="#msg-$msgId" aria-expanded="true">
+          <a>$messageName: $message.split("\r\n|\r|\n")[0]</a>
```

Comment:
Logic moved to Result class. First line of error message contains class and text of exception.